cmd/ocsp-updater/main.go

```diff
@@ -129,11 +140,63 @@ func newUpdater(
 		})
 	}
 
-	updater.ocspMinTimeToExpiry = config.OCSPMinTimeToExpiry.Duration
+	if config.AkamaiBaseURL != "" {
```

Comment:
(I'm inclined to do so because they are *also* going to have to make this code work for their CDN that they put in place for OCSP responses.)